rollup-common.js

```diff
@@ -4,7 +4,7 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-import summary from 'rollup-plugin-summary';
+import {summary} from 'rollup-plugin-summary';
```

Comment:
Why is this necessary? This seems out of scope for this PR?

Reply:
Adding the example projects (specifically the preact project with vite) nested under `packabes/labs/react` directory created `packages/labs/node_modules` that included rollup v3. The way rollup handles config files in v3 requires this change for the `build:rollup` script for `labs/react` package. This thankfully still works with other packages' rollup v2. I considered updating our whole repo to use rollup v3 but since this works for now, decided not to.

Reply:
Ah! Makes sense! Also yay for backwards compatibility :D